spec/decorators/course_decorator_spec.rb

```diff
@@ -596,6 +596,40 @@
     end
   end
 
+  describe "#academic_year" do
```

Comment:
Can we have the context of the `year` of the course start date...

It read deceptive when 2022 is omitted since that was skipped and the time frame used for testing spans 23 months as well as the test maybe using invalid options.

Given that there is only 22 months that is available to choice from.

Reply:
Done, I think

Reply:
Although I thought the idea of using the decorator was to avoid adding presentational methods to the model?

Reply:
This decorator is a bit over the top and @darokel wanna move away from it